docs/guides/avs-operator-guide.mdx

```diff
@@ -68,4 +68,4 @@ Now any Hyperlane message receiver can now leverage your validator signatures, h
 
 ## Learn more
 
-You can read more about the AVS protocol design [here](../protocol/eigenlayer-avs.mdx).
+You can read more about the AVS protocol design [here](/docs/protocol/economic-security/eigenlayer-avs.mdx).
```

Comment:
why make these absolute and not relative links?

Reply:
Less build failures if you had to move the current page